src/views/product-integrations-landing/components/tag-list/index.tsx

```diff
@@ -1,6 +1,22 @@
+import { IconArchive16 } from '@hashicorp/flight-icons/svg-react/archive-16'
+import { IconHandshake16 } from '@hashicorp/flight-icons/svg-react/handshake-16'
+import { IconHashicorp16 } from '@hashicorp/flight-icons/svg-react/hashicorp-16'
+import { IconRocket16 } from '@hashicorp/flight-icons/svg-react/rocket-16'
+import { IconWrench16 } from '@hashicorp/flight-icons/svg-react/wrench-16'
+import classNames from 'classnames'
 import Tooltip from 'components/tooltip'
+import {
+	Flag,
+	Integration,
+	Tier,
+} from 'lib/integrations-api-client/integration'
 import s from './style.module.css'
 
+export enum Size {
+	SMALL = 'small',
+	MEDIUM = 'medium',
+}
```

Comment:
Nit: This enum feels like an unnecessary hinderance on the consumer, since they need to import it too if they want to avoid type errors with the `TagListProps.size`.

A more friendly interface, (IMO and especially for components), would be to make this a string union:

```typescript
export type Size = "small" | "medium"
```

I think the compile-time type safety is comparable as well

Reply:
Oh yeah I'll adjust!